### ISSUE TITLE
document & distribute any core CSS dependencies

### ISSUE THREAD

**Issue by avkvirtru** [avkvirtru/virtuoso-design-system#59]
# 🚀 feature request

## Description
Virtuoso Design System expects default styles, most notably `font-size: 62.5%` on `html`. This is because [most of Virtru's front-end code uses that](https://github.com/search?l=&p=1&q=org%3Avirtru+%22html+%7B+font-size%3A+%2A%25%22+extension%3Acss+extension%3Ascss+-repo%3Avirtru%2FPDFTron&ref=advsearch&type=Code) as the base font size. 

## Describe the solution you'd like

1. Discuss core dependencies with team.

    1. base font-size
    1. reset or normalize styles
    1. box-sizing
    1. default margins or padding
    1. any styles generally defined on `html`, `body`, or `*` (e.g. disc from [etheria](https://github.com/virtru/etheria/pull/96/files#diff-4a0a792dbe56fd51b06751e5bdc269ed))

1. Assumptions like base font-size, along with things like reset or normalize CSS, should be at least documented in Storybook.

1. Ideally, these assumptions are distributed in a **base.css** spreadsheet.

## Describe alternatives you've considered
We could let every project that uses Virtuoso decide this themselves, but that feels like it's ripe for error and inconsistency.

**Comment by jrschumacher** [comment]
Do you think that we should apply resets if you use Virtuoso? https://dev.to/hankchizljaw/a-modern-css-reset-6p3

**Comment by jrschumacher** [comment]
Currently I only applied this to if you use the `<Layout>` component. This would be problematic for Dashboard and such.

**Comment by avkvirtru** [comment]
Yeah, great question. Precarious to build based on resets if not every project can import them. I would discuss as a team. 

Another approach may be to use `all: unset;` within each component's styles. [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/all).

**Comment by jrschumacher** [comment]
`all:reset` isn't supported in IE11. Also does this reset to nothing or to browser defaults. The docs aren't clear.
